eng/sendToHelix.proj

```diff
@@ -12,7 +12,7 @@
 
     <IncludeDotNetCli>true</IncludeDotNetCli>
     <DotNetCliPackageType>sdk</DotNetCliPackageType>
-    <DotNetCliVersion>3.1.405</DotNetCliVersion>
+    <DotNetCliVersion>6.0.400</DotNetCliVersion>
```

Comment:
I don't think that you can make the changes to this file like you are doing here. This depends on on some Helix infrastructure, so we should make sure which version can be used.